src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -967,6 +967,33 @@ function random_configuration_model(
     end
     return g
 end
+"""
+    uniform_tree(n)
+
+Generates a random labelled tree, drawn uniformly at random over the ``n^{n-2}`` such trees. A uniform word of length `n-2` over the alphabet `1:n` is generated (Prüfer sequence) then decoded. See also the `prufer_decode` function and [this page on Prüfer codes](https://en.wikipedia.org/wiki/Pr%C3%BCfer_sequence). 
+
+### Optional Arguments
+- `rng=nothing`: set the Random Number Generator.
+- `seed=nothing`: set the RNG seed.
+
+# Examples
+```jldoctest
+julia> uniform_tree(10)
+{10, 9} undirected simple Int64 graph
+```
+"""
+function uniform_tree(
+    n::Integer;
+    rng::Union{Nothing,AbstractRNG}=nothing,
+    seed::Union{Nothing,Integer}=nothing,
```

Comment:
I would remove the `seed` kwarg here - the other random generators just have this for legacy reasons, as we wanted to switch from  `seed` to `rng` but did not want to break existing code. Using `rng` instead of `seed` is consistent with other functions from the Julia standard library.

Reply:
That is ususally a good strategy, just that  some of the code here is rather old, but we can't always fix it.

Reply:
I removed `seed` from the arg and docstring. Is `rng = rng = rng_from_rng_or_seed(rng)` still necessary ?

Reply:
I would still use it, because that function then returns a default random generator, if that argument is omitted.